cmd/nodecmd/create.go

```diff
@@ -650,31 +680,38 @@ func setCloudInstanceType(cloudService string) (string, error) {
 	return nodeType, nil
 }
 
-func printResults(cloudConfigMap models.CloudConfig, publicIPMap map[string]string, ansibleHostIDs []string) {
+func printResults(cloudConfigMap models.CloudConfig, publicIPMap map[string]string, rpcNodeIPMap map[string]string, ansibleHostIDs []string) {
+	ux.Logger.PrintToUser("")
 	ux.Logger.PrintToUser("======================================")
 	ux.Logger.PrintToUser("AVALANCHE NODE(S) SUCCESSFULLY SET UP!")
 	ux.Logger.PrintToUser("======================================")
 	ux.Logger.PrintToUser("Please wait until the node(s) are successfully bootstrapped to run further commands on the node(s)")
 	ux.Logger.PrintToUser("")
+	if len(rpcNodeIPMap) > 0 {
+		ux.Logger.PrintToUser("")
+		ux.Logger.PrintToUser("======================================")
+		ux.Logger.PrintToUser("Regional RPC Endpoints: ")
+		for _, rpcNode := range rpcNodeIPMap {
+			ux.Logger.PrintToUser("    http://%s:9650", rpcNode)
+		}
+		ux.Logger.PrintToUser("======================================")
+		ux.Logger.PrintToUser("")
+	}
 	ux.Logger.PrintToUser("Here are the details of the set up node(s): ")
 	for region, cloudConfig := range cloudConfigMap {
-		ux.Logger.PrintToUser(fmt.Sprintf("Don't delete or replace your ssh private key file at %s as you won't be able to access your cloud server without it", cloudConfig.CertFilePath))
+		ux.Logger.PrintToUser("Don't delete or replace your ssh private key file at %s as you won't be able to access your cloud server without it", cloudConfig.CertFilePath)
+		ux.Logger.PrintToUser("Region: [%s] ", region)
 		for i, instanceID := range cloudConfig.InstanceIDs {
-			publicIP := ""
-			publicIP = publicIPMap[instanceID]
 			ux.Logger.PrintToUser("======================================")
-			ux.Logger.PrintToUser(fmt.Sprintf("Node %s details: ", ansibleHostIDs[i]))
-			ux.Logger.PrintToUser(fmt.Sprintf("Cloud Instance ID: %s", instanceID))
-			ux.Logger.PrintToUser(fmt.Sprintf("Public IP: %s", publicIP))
-			ux.Logger.PrintToUser(fmt.Sprintf("Cloud Region: %s", region))
+			ux.Logger.PrintToUser("Node %s details: ", ansibleHostIDs[i])
+			ux.Logger.PrintToUser("Cloud Instance ID: %s", instanceID)
+			ux.Logger.PrintToUser("Public IP: %s", publicIPMap[instanceID])
```

Comment:
probably indicate here if it an API node

Reply:
added